### PR TITLE
docs: fixed copy format and command

### PR DESCRIPTION
**Pull request by MrSagarShah**
## Status

**READY**

## Description

- Removed the comment from the copiable command 
- Update the title of the command 
- Fixed the command - was missing ending '"' 

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [X] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,10 +26,10 @@ This template includes a simple demo game with the basics you'll need for game d
 dart pub global activate very_good_cli
 ```
 
-2. Generate your game
+2. Generate your game named my_game
```
Comment (tomarra) [inline README.md:29]: I think we revert this title change and put the comment back in to describe the name parameter. Let's focus the fix on the missing `"` which is a good catch.

Comment (alestiago) [inline README.md:29]:
```suggestion
2. Generate your game
```

File: README.md [hunk 2]
```diff
@@ -27,9 +27,9 @@ dart pub global activate very_good_cli
 ```
 
 2. Generate your game
+
 ```sh
-# Create a new Flame game named my_game
```
Comment (alestiago) [inline README.md:31]:
```suggestion
# Create a new Flame game named my_game
```

File: README.md [hunk 3]
```diff
@@ -27,9 +27,9 @@ dart pub global activate very_good_cli
...
-very_good create flame_game my_game --desc "My new Flame game
+very_good create flame_game my_game --desc "My new Flame game"
```
Comment (alestiago) [inline README.md:32]:
```suggestion
# Create a new Flame game named my_game
very_good create flame_game my_game --desc "My new Flame game"
```

**Review by alestiago — APPROVED** [review]

**Pull request merged**